packages/vscode-extension/CHANGELOG.md

```diff
@@ -1,9 +1,75 @@
-# Change Log
+# Changelog
 
-All notable changes to the "teamsfx-toolkit" extension will be documented in this file.
+## 2.0.0 - May 25 2021
+2.0.0 is a major version upgrade for Teams Toolkit with many new improvements and supports.
+- Add: new design of UI and command palette list
+- Add: new getting started guide, samples and doc link from toolkit
+- Add: new sign-in experience for M365 and Azure
+- Add: new interactive flow for screating new app
+- Add: add message extension capability to Teams app
+- Add: sample codes for in-meeting app.
+- Add: add Typescript support
+- Add: support e2e dev experience for tab app hosted by either Azure or SPFx
+- Add: support adding backend API (Azure Functions) and SQL DB to tab app
+- Add: remote debugging support for tab app, bot app, and message extension
```

Comment:
Let's not say "remote debugging", but use "preview"

Reply:
will fix. Long found an issue that cannot merge, so will fix all and create a new PR.